language/stdlib/modules/Diem.move

```diff
@@ -406,9 +415,15 @@ module Diem {
         preburn_address: address,
```

Comment:
passing in mutable preburn, so can be untouched then have another function which adds the preburn to the concurrentresource